Added in basic styling for first few pages

sarrahzali opened the pull request.

Commit:
Styled all of the first few pages

Commit:
Styled all of the first few pages

Commit:
Merge branch 'sarrahzalidesign' of https://github.com/Get-Your-GreenBack-Tompkins/gygb-client-web into sarrahzalidesign

ewlsh commented on src/assets/try.svg:
```diff
@@ -0,0 +1,13 @@
+<svg width="375" height="812" viewBox="0 0 375 812" fill="none" xmlns="http://www.w3.org/2000/svg">
```
We should split the top/bottom lines into separate SVGs.

ewlsh commented on src/pages/Correct.tsx:
```diff
@@ -29,9 +38,10 @@ const generateContinueButton = (questionNum: number, setQuestionNum: Function, q
     <IonPage>
-      <IonContent fullscreen class="ion-padding">
+      <IonContent fullscreen>
         <IonToolbar>
```
This should be converted into an `IonGrid` structure instead of a toolbar.

ewlsh commented on src/assets/logo.svg:
```diff
@@ -0,0 +1,10 @@
+<svg width="405" height="244" viewBox="0 0 405 244" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
This is actually an SVG file with an embedded PNG file. We should avoid that due to potential issues and also keep the text separate to prevent breakage (screenreaders, image compression, etc)

ewlsh commented on src/assets/rLine.svg:
```diff
@@ -0,0 +1,51 @@
+<svg width="414" height="505" viewBox="0 0 414 505" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Is this used somewhere?

ewlsh commented on src/pages/Correct.tsx:
```diff
@@ -1,6 +1,15 @@
+  IonImg,
+  IonTitle } from '@ionic/react';
 import { RouteComponentProps } from 'react-router';
+import greenLines from "../assets/greenLines.svg";
```
Capitalize: `GreenLines`

ewlsh commented on src/pages/Incorrect.tsx:
```diff
@@ -1,6 +1,7 @@
-import { IonPage, IonButton, IonContent, IonToolbar, IonTitle } from '@ionic/react';
+import { IonPage, IonButton, IonContent, IonToolbar, IonTitle, IonImg } from '@ionic/react';
 import { RouteComponentProps } from 'react-router';
+import redLines from "../assets/redLines.svg";
```
Capitalize: `RedLines`

ewlsh commented on src/pages/Incorrect.tsx:
```diff
@@ -15,23 +16,25 @@ const generateSkipButton = (questionNum: number, setQuestionNum: Function, quiz:
     <IonPage>
-      <IonContent fullscreen class="ion-padding">
+      <IonContent fullscreen>
         <IonToolbar>
```
Use `IonGrid` and `hx` instead of `IonToolbar` and `IonTitle`

ewlsh commented on src/pages/Quiz.tsx:
```diff
@@ -9,7 +9,9 @@ import {
 } from "@ionic/react";
 import { RouteComponentProps } from "react-router";
-import WelcomePath from "../assets/welcome-path.svg";
+// import WelcomePath from "../assets/welcome-path.svg";
```
Remove comment.

ewlsh commented on src/pages/Quiz.tsx:
```diff
@@ -18,25 +20,26 @@ interface Props extends RouteComponentProps {
   return (
     <IonPage>
-      <IonContent fullscreen class="ion-padding">
+        
```
No new line.

ewlsh commented on src/pages/Quiz.tsx:
```diff
@@ -18,25 +20,26 @@ interface Props extends RouteComponentProps {
+        <IonImg className="home-lines" src={Try}></IonImg>
+        <IonImg className="logo" src={Logo}></IonImg>
+        <IonButton size="large" className="welcome-button" routerLink="/quiz/question">
+                WELCOME
```
Avoid capitalizing text. Instead, use `text-transform: capitalize` in CSS.

ewlsh commented on src/pages/Quiz.tsx:
```diff
@@ -18,25 +20,26 @@ interface Props extends RouteComponentProps {
+        
+        <IonImg className="home-lines" src={Try}></IonImg>
+        <IonImg className="logo" src={Logo}></IonImg>
+        <IonButton size="large" className="welcome-button" routerLink="/quiz/question">
```
Move this into an `IonGrid`

ewlsh commented on src/pages/Quiz.tsx:
```diff
@@ -18,25 +20,26 @@ interface Props extends RouteComponentProps {
           <IonRow>
             <IonCol>
-              <IonImg src="/assets/logo.png" />
               <h1 className="subtitle">{quiz && quiz.name}</h1>
```
This is the text which should be under the logo - right now it is hidden on most screen widths but will break the site layout on weird ratios/small widths.

ewlsh commented on src/theme/style.scss:
```diff
@@ -1,29 +1,81 @@
 
 body {
-  font-family: "Montserrat", "sans-serif";
+  font-family: "serif", "sans-serif";
```
I used `Montserrat` because it approximates Proxima Nova. This font set says "use serif font" (like Times New Roman), if there isn't a "serif" font use the default "sans-serif". There will never *not* be a serif font.

ewlsh commented on src/theme/style.scss:
```diff
@@ -1,29 +1,81 @@
 body {
-  font-family: "Montserrat", "sans-serif";
+  font-family: "serif", "sans-serif";
+  background-color: "#83CE27";
```
The body probably shouldn't have a green background color.

ewlsh commented on src/theme/style.scss:
```diff
@@ -1,29 +1,81 @@
+  background-color: "#83CE27";
 }
 
+svg{
```
Space before `{`

ewlsh commented on src/theme/style.scss:
```diff
@@ -1,29 +1,81 @@
 
+svg{
+ display: flex;
+ position: relative;
```
Indent should be `2`

ewlsh commented on src/theme/style.scss:
```diff
@@ -1,29 +1,81 @@
 
-.welcome-content {
+#root{
+  margin: 0px;
```
If you ever need to use 0 - and be careful setting it on roots - you should use `0` instead of `0px`, the unit doesn't matter and `0` is faster in some browsers.

ewlsh commented on src/theme/style.scss:
```diff
@@ -1,29 +1,81 @@
+}
+
+.star{
+  position: absolute;
```
This probably shouldn't be absolutely positioned, it should be able to fit into the page layout somehow.

ewlsh commented on src/theme/style.scss:
```diff
@@ -1,29 +1,81 @@
+  top:25%;
+  bottom: 25%;
+}
+.welcome-button{
```
Newlines needed.

ewlsh commented on src/theme/style.scss:
```diff
@@ -1,29 +1,81 @@
+  left: 25%;
+  right: 25%;
+  border-radius: 30px;
+  position: absolute;
```
This should also be able to fit into the layout.

ewlsh commented on src/theme/style.scss:
```diff
@@ -1,29 +1,81 @@
+  right: 3%;
+  left: 3%;
+  position: absolute;
+}
```
Newline and this should also be in-layout.

Commit:
Still gotta fix star

Commit:
Dont deep clean, still will fix

Commit:
Dont deep clean, still will fix

Commit:
Merge branch 'sarrahzalidesign' of https://github.com/Get-Your-GreenBack-Tompkins/gygb-client-web into sarrahzalidesign

ewlsh committed:
Quiz styling tweaks & fixes.

jw792 committed:
Update Quiz.tsx

Commit:
pulled from dev

Commit:
E

Commit:
styling not being applied, test again

ewlsh committed:
Remove extraneous red color.

ewlsh committed:
Format.

ewlsh committed:
Format code, cleanup imports.

ewlsh committed:
Split styling into multiple files, CSS cleanup.
- Use variables for common colors.
- Use vw/vh where applicable to avoid bugging out on thin phones.
- Cleanup SVG height/width.

ewlsh committed:
Separate welcome and tutorial.

ewlsh committed:
Merge branch 'tutorial' into sarrahzalidesign

ewlsh committed:
Fix initial link.
